_posts/2017-03-18-hypatia-meets-womens-energy-club_en.md

```diff
@@ -0,0 +1,49 @@
+
+[**Hypatia Frauennetzwerk Erneuerbare Energien und Cleantech e.V.**](www.hypatia-network.de) is a non-profit organization for women working in the renewable energy sector. Similar to the Women's Energy Club it aims to empower women and provide professional support. Increasing the visibility of women in the sector is one of the major goals to reach for professional equity for women and men. Hypatia has about 30 members. The organization team works on voluntary basis.
+
+The same objectives led the Women's Energy Club to invite Hypatia in order to present our network & benefit from each other knowledge.  
+As a representative of Hypatia I traveled to the latest meeting of the Club on 15th of March in Kiev. We established that the general problems women face in professional life are the same in Ukraine and Germany.  
+A lot of women are not aware of being discriminated in the professional context. Our networks want to increase awareness and change the established paradigms. This led to the conclusion to **give women the opportunity of choice without any disadvantages in the professional context**.  
+The main conclusions of our discussion are listed below:
+
+*General Problems*
+- Visibility of women in profession
+- Lack of female role-models
+- Inefficient knowledge exchange between women concerning career issues (salary, work conditions...)
+- Equity of men and women in profession (acceptance, opportunity, salary...) 
+
+*Importance of Networking*  
+- Knowledge exchange concerning career & work-life-balance
+- Support for women in male-dominated sectors
+- Open table discussions concerning career and/or related technology topics
+- Conduct workshops and events to support women's career with skills and strategies
+- Collect data concerning e.g. the working environment for women in the energy sector
+
+The meeting gave good incentives for further cooperation and made clear that women networks are as important as ever. 
+
+**Special thanks** goes to the amazing women who established the **Women's Energy Club**:  
+- Oksana Aliieva (Program Coordinator "Climate Change and Energy Policy Program" at the [Heinrich Boell Foundation Ukraine](http://ua.boell.org/en))  
+- Valentyna Beliakova (Director of the [European-Ukrainian Energy Agency](http://euea-energyagency.org/en/))  
+- Kateryna Babichok (Project Assistant at the [European-Ukrainian Energy Agency](http://euea-energyagency.org/en/))
+
+**Do you have any questions concerning this blog post and our networks? Please feel free contact us:**
+
+Hypatia – Frauennetzwerk Erneuerbare Energien und Cleantech e.V.  
+email: kontakt[at]hypatia-network.de  
+twitter: @hypatia-network
+
+Women's Energy Club Ukraine
+email:
+facebook: 
+
+[Hier geht's zur deutschen Version](Link)
```

Comment:
Instead of hiding this little snippet below the post, how about making this note more prominent and move it to the top?

I suggest to extend the text to something like

> Dieser Blogeintrag ist auch auf deutsch verfügbar. Du findest ihn im Hypathia Blog. [Hier geht's lang](#).